tarantool.go

```diff
@@ -53,16 +65,16 @@ func Validate(schema string) error {
 	// Setup the request
 	req, err := http.NewRequest(http.MethodPost, tarantoolURL, bytes.NewBuffer(reqBody))
 	if err != nil {
-		return err
+		return newServiceError(err)
 	}
 
 	// Call the Tarantool validator
 	res, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return err
+		return newServiceError(err)
```

Comment:
We should still distinguish between temporary `net.Error` vs other errors here.. So I would only return `ServiceError` when it's the aforementioned `net.Error.Temporary()` (feel free to disagree though).